test/tstQueryTreeCallbacks.cpp

```diff
@@ -103,7 +103,15 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(callback_spatial_predicate, TreeTypeTraits,
   auto values = initialize_values(points, /*delta*/ 0.f);
   std::vector<int> offsets = {0, n};
 
+#ifdef ARBORX_APIV1
   Tree const tree(ExecutionSpace{}, points);
+#else
+  Tree const tree(
+      ExecutionSpace{},
+      ArborX::Details::LegacyValues<decltype(points),
+                                    typename Tree::bounding_volume_type>{
+          points});
+#endif
```

Comment:
Did you consider wrapping the tree and defining a test-only class that construct with legacy values instead of explicitly converting?
That would probably get rid of a lot of boilerplate code.

Reply:
Can you please elaborate?